syntax = "proto3";

package pb;

option go_package = "./pb";

message RecommendationRequest {
  string user_id = 1;
}

message ProductReplica {
  string id = 1;
  string name = 2;
  string description = 3;
  double price = 4;
}

message RecommendationResponse {
  repeated ProductReplica recommended_products = 1;
}

service RecommenderService {
  rpc GetRecommendations (RecommendationRequest) returns (RecommendationResponse);
}
